### ISSUE TITLE
Unable to compile

### ISSUE THREAD

**Issue by guidedways** [zorgiepoo/sparkle-ui-xpc-issues#45]
Somehow `make release` worked the firs time. Thereafter I'm see the following on the console:

```
** BUILD FAILED **

The following build commands failed:
	CompileSwift normal x86_64
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
make: *** [release] Error 65
```

In addition to more errors:

```
/Sparkle-ui-separation-and-xpc/generate_appcast/Appcast.swift:11:22: error: use of undeclared type 'SUError'
func makeError(code: SUError, _ description: String) -> NSError {
```

**Comment by johnwbyrd** [comment]
Same problem here.  Looks like the maintainers not doing any build or soak tests on this branch.  This branch looks dead to me at the moment, which is a shame.

**Comment by zorgiepoo** [comment]
I think the branch is being more maintained on the official Sparkle repo. I'm not mainintaing it here currently. IIRC, though, the DerivedData build directory or something may need to be nuked from time to time for the tests as there was some issue there.

**Comment by zorgiepoo** [comment]
Yeah, please look at the read me of this repo. I'm not sure how ya'll are getting here!

**Comment by johnwbyrd** [comment]
The ui-separation-and-xpc branch of the main Sparkle repo links here: https://github.com/sparkle-project/Sparkle/tree/ui-separation-and-xpc

**Comment by johnwbyrd** [comment]
The ui-separation-and-xpc branch of the main Sparkle repo still links here.  What's going on on that branch vs. this one?  Which do we consider authoritative
